src/ray/common/id.h

```diff
@@ -45,6 +44,8 @@ class BaseID {
   std::string Hex() const;
 
  protected:
+  static T FromRandom();
```

Comment:
I think it's fine to leave `FromRandom` public.

Reply:
JobID needn't FromRandom I think, otherwise, other developers may use `JobID::FromRandom` unintentionally.

Reply:
you can only make `Job::FromRandom` private, or mark it `= delete`.

Reply:
Good idea!